Library/Formula/relx.rb

```diff
@@ -0,0 +1,43 @@
+class Relx < Formula
+  homepage "https://github.com/erlware/relx"
+  url "https://github.com/erlware/relx.git", :tag => "v1.2.0"
+
+  head "https://github.com/erlware/relx.git"
+
+  depends_on "erlang"
+  depends_on "rebar"
+
+  def install
+    system "make", "escript"
+    bin.install "relx"
```

Comment:
It'd be good if we could work out a way to not have to clone a Git repository which is much slower and doesn't give us specific version checksums. Thanks!